### DISCUSSION TITLE
Saving hasMany item gives error

### DISCUSSION THREAD

**Discussion by Glenf** [statamic-rad-pack/runway#266] [Help]
Hello again,

When I try to save hasMany related child item, it gives me following error:
`Call to undefined method /App/Models/ChildModelName::parent_item_id()`

Saving in it self works correctly. I tried to follow this up and things point to here: https://github.com/duncanmcclean/runway/blob/27d1850a5b0a3baeeee55606bcc2b4cb389959ea/src/Http/Controllers/ResourceController.php#L298

I'll build you an example project here: https://github.com/Glenf/runway-hasMany-issue

Steps to reproduce (with example project):
1. Open Statamic CP
2. Click 'Parents' in the sidebar
3. Select a parent
4. Select a child under the parent
5. Click 'Save' on child

**Comment by duncanmcclean** [comment]
I've moved this in to an issue since it sounds like a bug: https://github.com/duncanmcclean/runway/issues/268
